src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ApplicationsController.kt

```diff
@@ -407,31 +404,20 @@ class ApplicationsController(
   override fun applicationsApplicationIdDocumentsGet(applicationId: UUID): ResponseEntity<List<Document>> {
     val deliusPrincipal = httpAuthService.getDeliusPrincipalOrThrow()
     val username = deliusPrincipal.name
-
     val application = extractEntityFromCasResult(applicationService.getApplicationForUsername(applicationId, username))
 
-    val documentsFromApDeliusFlag = featureFlagService.getBooleanFlag("get-documents-from-ap-delius")
-    val documents = when (val documentsResult = getDocuments(isDocumentFromAPDelius = documentsFromApDeliusFlag, application.crn)) {
-      is AuthorisableActionResult.NotFound -> throw NotFoundProblem(application.crn, "Person")
-      is AuthorisableActionResult.Unauthorised -> throw ForbiddenProblem()
-      is AuthorisableActionResult.Success -> documentsResult.entity
-    }
-
-    val transformedDocuments = when (documents) {
-      is GroupedDocuments -> documentTransformer.transformToApi(documents)
-      is List<*> -> documentTransformer.transformToApi(
-        documents.filterIsInstance<APDeliusDocument>(),
-      )
-      else -> emptyList()
-    }
+    val documentsResult = getDocuments(application.crn)
+    val apiDocuments = documentTransformer.transformToApi(documentsResult)
 
-    return ResponseEntity(transformedDocuments, HttpStatus.OK)
+    return ResponseEntity(apiDocuments, HttpStatus.OK)
   }
 
-  private fun getDocuments(isDocumentFromAPDelius: Boolean, crn: String) = if (isDocumentFromAPDelius) {
-    offenderService.getDocumentsFromApDeliusApi(crn)
-  } else {
-    offenderService.getDocumentsFromCommunityApi(crn)
```

Comment:
I think this function can now be removed?

`fun getDocumentsFromCommunityApi(crn: String): AuthorisableActionResult<GroupedDocuments>`